### DISCUSSION TITLE
how to pass options to `remark-rehype`?

### DISCUSSION THREAD

**Discussion by stefanprobst** [mdx-js/mdx#1934] [Q&A] [answered]
hi, i would like to pass [`footnoteLabel`](https://github.com/remarkjs/remark-rehype#optionsfootnotelabel) and [`footnoteBackLabel`](https://github.com/remarkjs/remark-rehype#optionsfootnotebacklabel) options to `remark-rehype` - is this currently possible? looking at [this line](https://github.com/mdx-js/mdx/blob/71bc6ff9b4676e3f056266a04966b01a9f3e6216/packages/mdx/lib/core.js#L106), it seems it might not?

**Comment by wooorm** [comment, marked as answer]
Not possible yet, no. But would be useful. Named `remarkRehypeOptions` (similar to [`react-markdown`](https://github.com/remarkjs/react-markdown#props)). Important to always set `allowDangerousHtml: true`, and merge `passThrough` properly?
